projects/igniteui-angular/src/lib/query-builder/query-builder.component.spec.ts

```diff
@@ -2043,6 +2042,204 @@ describe('IgxQueryBuilder', () => {
             QueryBuilderFunctions.verifyEditModeExpressionInputValues(fix, 'OrderId', 'In', '', 0);
             QueryBuilderFunctions.verifyEditModeExpressionInputValues(fix, 'Released', 'True', '', 1);
         }));
+
+        it(`Should apply 'string' column validators properly and calculate 'canCommit' base on it.`, fakeAsync(() => {
```

Comment:
I beleive that simply checking the props/attributes are applied on the inputs/pickers should be enough of a test. The actual effect of these props should have been tested in the inputs themselves.